simplecov-html.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rake'
   s.add_development_dependency 'sprockets'
   s.add_development_dependency 'sass'
+  s.add_development_dependency 'test-unit' if RUBY_VERSION >= '2.2.0'
```

Comment:
hmm,  normally I'm :-1: dynamic dependency in a gemspec, but since it's dev, I guess it doesn't really matter. No one runs [gem test](https://github.com/rubygems/rubygems-test) any more.